common/djangoapps/student/management/commands/set_superuser.py

```diff
@@ -0,0 +1,46 @@
+"""Management command to grant or revoke superuser access for one or more users"""
+
+from optparse import make_option
+from django.contrib.auth.models import User
+from django.core.management.base import BaseCommand, CommandError
+
+
+class Command(BaseCommand):
+    """Management command to grant or revoke superuser access for one or more users"""
+    option_list = BaseCommand.option_list + (
+        make_option('--unset',
+                    action='store_true',
+                    dest='unset',
+                    default=False,
+                    help='Set is_superuser to False instead of True'),
+    )
+
+    args = '<user|email> [user|email ...]>'
+    help = """
+    This command will set is_superuser to true for one or more users.
+    Lookup by username or email address, assumes usernames
+    do not look like email addresses.
+    """
+
+    def handle(self, *args, **options):
+        if len(args) < 1:
+            raise CommandError('Usage is set_superuser {0}'.format(self.args))
+
+        for user in args:
+            try:
+                if '@' in user:
+                    userobj = User.objects.get(email=user)
+                else:
+                    userobj = User.objects.get(username=user)
+            except:
+                print "Could not get user with identifier " + user
+                raise
```

Comment:
Just as a nit, I'm not certain you actually want to raise the error here, do you? If you pass in a list of 100 users, and the 10th one errors, you still have to basically reprocess the whole list. I was more thinking you'd try to process them all, and print messages for the few that didn't work. So you'd process all 100 and just see

```
Could not get user with identifier mxa
Could not get user with identifier sarin
```
and be like, oh duh, I misspelled 'max' and 'sarina', I'll rerun those two.